# 프로그램 시작
print("# 프로그램이 시작되었습니다!")

# 예외 발생 코드 해결
list_a = [1, 2, 3, 4, 5]
list_a[1]
